tests/network/primary_pod_network.go

```diff
@@ -178,13 +178,30 @@ func vmiWithMasqueradeBinding() *v1.VirtualMachineInstance {
 	return vmi
 }
 
-func newFedoraWithGuestAgent() (*v1.VirtualMachineInstance, error) {
+func newFedoraWithGuestAgentAndMasqueradeBinding() (*v1.VirtualMachineInstance, error) {
 	networkData, err := libnet.CreateDefaultCloudInitNetworkData()
 	if err != nil {
 		return nil, err
 	}
 
 	tmpVmi := libvmi.NewFedora(
+		libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+		libvmi.WithNetwork(v1.DefaultPodNetwork()),
+		libvmi.WithCloudInitNoCloudUserData(tests.GetGuestAgentUserData(), false),
```

Comment:
Why not use the SRIOV fedora image? It includes it.

Reply:
I am worry that if we add the fedora SRIOV image something get added in the future to fix SRIOV lane or the like that breaks the non SRIOV image, but if we are really sure that this is not going to happened I am ok with it, or even better create a new fedora-ga image and use it at the end is all we need.

Reply:
It was badly named, it should have been called `fedora-extended` or something similar.
Here is its [content](https://github.com/kubevirt/kubevirtci/blob/master/cluster-provision/images/vm-image-builder/fedora-sriov-lane/cloud-config).

But if you prefer not to use it now, I won't push. We can change this in the future as well.

Reply:
I prefer not to right now, rename the image or create new one and do the change.